profiler/profiler.go

```diff
@@ -261,6 +262,7 @@ func start(cfg Config, options ...option.ClientOption) error {
 
 func debugLog(format string, e ...interface{}) {
 	if config.DebugLogging {
+		log.SetPrefix("Cloud Profiler Go Agent: ")
```

Comment:
This will set the prefix for all messages logged through standard log package. Plus there is no need to call this on every message. I think you want something like https://play.golang.org/p/BSwVA65zlGU. Or manually add the prefix for each line in the logged string.

Reply:
Good catch, thanks. Fixed.